### PR TITLE
docs: logger: add logger usage description

### PR DESCRIPTION
**Pull request by bkokoszx**
I've added logger usage description. It is actually ported README tools file.

Signed-off-by: Bartosz Kokoszko <bartoszx.kokoszko@linux.intel.com>

### CODE REVIEW

**Comment by bkokoszx** [comment]
I would be thankful for every style suggestions - it is my first time using rst.

**Comment by jajanusz** [comment]
looks good

File: developer_guides/debugability/logger/index.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+
+Examples:
+
+- Get traces from "/sys/kernel/debug/sof/etrace" file, verifies fw_version with
+  "/sys/kernel/debug/sof/fw_version" and prints logs to stdout 
+ 
+.. code-block:: bash
+
+	$ sof-logger -l ldc_file -e
```
Comment (jajanusz) [inline developer_guides/debugability/logger/index.rst]: Just one thing - indentation is 3 spaces not tab

Comment (jajanusz) [inline developer_guides/debugability/logger/index.rst]: Just one thing - indentation should be 3 spaces instead of tab

Comment (bkokoszx) [reply developer_guides/debugability/logger/index.rst]: updated

**Pull request merged**